main.c

```diff
@@ -490,6 +516,7 @@ static void parse_args(int argc, char *const argv[], struct options *options)
         { "outbound-addr", required_argument, NULL, OUTBOUND_ADDR },
         { "outbound-addr6", required_argument, NULL, OUTBOUND_ADDR6 },
         { "disable-dns", no_argument, NULL, DISABLE_DNS },
+        { "hwaddress", required_argument, NULL, HWADDRESS },
```

Comment:
I'd personally prefer to call it `mac` rather than `hwaddress`, because it is actually software :)

Reply:
Sure, I will make that change.

Reply:
This has been done, resolving conversation.